cmd/imagec/docker.go

```diff
@@ -55,11 +61,14 @@ type Manifest struct {
 
 // V1Compatibility represents some parts of V1Compatibility
 type V1Compatibility struct {
-	ID        string    `json:"id"`
-	Parent    string    `json:"parent,omitempty"`
-	Comment   string    `json:"comment,omitempty"`
-	Created   time.Time `json:"created"`
-	Container string    `json:"container,omitempty"`
+	docker.V1Image
```

Comment:
@caglar10ur good call, how's `ImageConfig`? I cringed at the thought of putting "metadata" in the name, and docker calls it "ImageConfigJSON" in documentation that I've seen.